### PR TITLE
Show key concept on the level details dialog

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Resolves [PLAT-919](https://codedotorg.atlassian.net/browse/PLAT-919). The key concept is shown in the curriculum builder pull through, so we should show it here too.

Screen Shot 2021-03-30 at 3 40 26 PM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/instructions/TeacherFeedback.jsx [hunk 1]
```diff
@@ -419,9 +421,7 @@ export class TeacherFeedback extends Component {
 export const UnconnectedTeacherFeedback = TeacherFeedback;
 export default connect(state => ({
   viewAs: state.viewAs,
-  serverScriptId: state.pageConstants.serverScriptId,
```
Comment (dmcavoy) [inline apps/src/templates/instructions/TeacherFeedback.jsx:422]: Did you mean to remove these? Are they just unused and this is clean up?

Comment (dmcavoy) [reply apps/src/templates/instructions/TeacherFeedback.jsx:422]: Oh looks like they are just not coming from redux anymore

**Comment by dmcavoy** [comment]
All the changes look fine but can I ask why this is what was needed to make the key concept show?

File: apps/src/templates/instructions/TeacherFeedback.jsx (hunk 1 above)
Comment (bethanyaconnor) [reply apps/src/templates/instructions/TeacherFeedback.jsx:422]: Yeah, I just passed them through. They were already being accessed in `TopInstructions` anyway.

**Comment by bethanyaconnor** [comment]
@dmcavoy Turns out I just needed to pass the level id through so that [this code](https://github.com/code-dot-org/code-dot-org/blob/c23ed8ad4905c21ac397c96a530fb94cb01ae995/apps/src/templates/instructions/TopInstructions.jsx#L225) would be run

**Review by dmcavoy — APPROVED** [review]

**Pull request merged**